tests/countries/test_guam.py

```
@@ -55,3 +47,62 @@ def test_government_holidays(self):
             ("2024-12-25", "Christmas Day"),
         )
+
+    def test_unofficial_holidays(self):
+        self.assertHolidays(
+            HolidaysGU(categories=UNOFFICIAL, years=2024),
+            ("2024-02-14", "Valentine's Day"),
+            ("2024-03-17", "Saint Patrick's Day"),
+            ("2024-10-31", "Halloween"),
+        )
+
+    def test_l10n_default(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)